controllers/suite_test.go

```diff
@@ -81,9 +81,10 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(err).ToNot(HaveOccurred())
 
 	err = (&ManagedOCSReconciler{
-		Client: k8sManager.GetClient(),
-		Log:    ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
-		Scheme: scheme.Scheme,
+		Client:           k8sManager.GetClient(),
+		Log:              ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
+		Scheme:           scheme.Scheme,
+		ConfigSecretName: "test-secret",
```

Comment:
I would suggest moving the value into a public const at the top of the file to be used both here and on every test suite where it is needed.

Reply:
Done